perf/istio-install/setup_istio.sh

```diff
@@ -141,7 +141,16 @@ function install_istio_with_helm() {
 # install istio with default IstioControlPlane CR yaml using istioctl.
 function install_istio_with_istioctl() {
   local CR_PATH="${WD}/istioctl_profiles/${CR_FILENAME}"
+  pushd "${ISTIOCTL_PATH}"
   ./istioctl manifest apply -f "${CR_PATH}" --set "${SET_OVERLAY}" "${EXTRA_ARGS}"
+  popd
+}
+
+function install_telemetryv2_stats() {
+  export SET_OVERLAY="values.telemetry.enabled=true,values.telemetry.v2.enabled=true"
```

Comment:
@mandarjog Does this command will enable both metadata exchange and stats filter? If so, how can we separately test the affect of metadata exchange?

Reply:
this is equivalent to existing setup(with metadata exchange and stats filter), I will add stackdriver related setup in another PR